core/traits/manual_scaler_trait.md

```diff
@@ -0,0 +1,89 @@
+# Manual Scaler
+
+This trait definition allows operators to manually scale the number of replicas for workload types that allow scaling operations.
+
+## Applicable Workloads
+
+ - `core.oam.dev/v1alpha2.ContainerizedWorkload`
+
+## Schematic
+
+| Attribute | Type | Required | Default Value | Description |
+|-----------|------|----------|---------------|-------------|
+| `replicaCount` | `integer` | Y | `0` | The target number of replicas to create for a given component instance. |
+
+## Spec
+
+As a core trait, we don't need any trait definition here. But we could give this as an example.
+
+Firstly, we need register trait definition like below:
+
+```yaml
+apiVersion: core.oam.dev/v1alpha2
+kind: TraitDefinition
+metadata:
+  name: manualscalertrait.core.oam.dev
+spec:
+  appliesToWorkloads:
+    - core.oam.dev/v1alpha2.ContainerizedWorkload
+  definitionRef:
+    name: manualscalertrait.core.oam.dev
+```
+
+Then OAM runtime should have a version of this schema, the schema SHOULD be able to validate. For example using JSON Schema like this:
```

Comment:
Maybe add a CRD as example as well?

Reply:
Perhaps a CRD-compatible OpenAPIv3 schema? This would make it easy to relate the schema to a Kubernetes CRD without leaking concrete Kubernetes specifics into the spec.

Reply:
OK I'll add a CRD-compatible OpenAPIv3 schema